cmd/registry.go

```diff
@@ -124,9 +124,13 @@ func parseInfo(item string) (string, string, error) {
 	parts := strings.Split(item, "=")
 
 	if len(parts) != 2 {
-		return "", "", fmt.Errorf(`%s is invalid, Must be in format key=value
+		return "", "", fmt.Errorf(`%s is invalid. Must be in format key=value
 Examples: username=bob password=s3cur3pw1`, item)
 	}
 
+	if parts[0] != "username" && parts[0] != "password" {
```

Comment:
We'll just have to remember to update this when `hostname` (will be needed for Builder in the future) and potentially other items become available